From 7b2bd676775714ca9e00ad09c5bfdc777fcf1ec4 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Mon, 16 Nov 2020 16:06:18 +0800
Subject: [PATCH] arm64: dts: rockchip: Remove HS400 support from RK356X
 platforms

It turns out to be that RK356X platforms is only support to work
reliably under HS200 mode. We now officially remove HS400
support from datasheet, as well as dts.

Change-Id: Icb7cff46a6d757d1615360c04f4b20b3a5bdd6d2
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet.dts | 6 ++----
 arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi         | 3 +--
 2 files changed, 3 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet.dts b/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet.dts
index 712c5793f448..d6584dbf5828 100644
--- a/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3566-rk817-tablet.dts
@@ -891,13 +891,11 @@
 
 &sdhci {
 	bus-width = <8>;
-	mmc-hs400-1_8v;
 	supports-emmc;
 	non-removable;
 	keep-power-in-suspend;
-	rockchip,txclk-tapnum = <0xa>;
-	max-frequency = <100000000>;
-	mmc-hs400-enhanced-strobe;
+	rockchip,txclk-tapnum = <0x10>;
+	max-frequency = <200000000>;
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi
index d315a4d3e4af..4ebc98cb1188 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi
@@ -1511,12 +1511,11 @@
 
 &sdhci {
 	bus-width = <8>;
-	mmc-hs400-1_8v;
 	supports-emmc;
 	non-removable;
 	keep-power-in-suspend;
+	rockchip,txclk-tapnum = <0x10>;
 	max-frequency = <200000000>;
-	mmc-hs400-enhanced-strobe;
 	status = "okay";
 };
 
-- 
2.35.3

